cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1374,15 +1374,6 @@ def test_advanced_components(self):
         self.assertNotEqual(only_template.get('category'), 'video')
         self.assertNotEqual(only_template.get('category'), 'openassessment')
 
-    def test_advanced_components_without_display_name(self):
```

Comment:
I think I'd rather get rid of this for now and re-introduce it if we write/make any other components that need to make use of this functionality.